src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/PresentationBuildTasks.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net472;net8.0</TargetFrameworks>
+    <TargetFrameworks>net472;net481;net8.0</TargetFrameworks>
```

Comment:
Are we going to need to produce our own targets/targets and not use the ones backed in the FX?

Reply:
That's a good question - I think there are some changes to this codebase that may require compensating changes to build tasks/targets.  Could be that we want forked SDK stuff too.